setup.py

```diff
@@ -37,7 +37,7 @@ def read(fname):
 
 
 aws_deps = ['boto3']
-gcs_deps = ['google-cloud-storage>=1.31.0']
+gcs_deps = ['google-cloud-storage>=2.6.0']
```

Comment:
for future reference, this version bump relates to https://github.com/googleapis/python-storage/pull/878 and the general maturing of the `open` interface in v2